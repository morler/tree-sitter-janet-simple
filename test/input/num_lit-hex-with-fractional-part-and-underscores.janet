0x0_9_.1_f__