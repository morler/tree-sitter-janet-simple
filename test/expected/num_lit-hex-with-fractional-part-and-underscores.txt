(source [0, 0] - [0, 12]
  (num_lit [0, 0] - [0, 12]))
